### DISCUSSION TITLE
Bug report: Send Later panel still blank in Thunderbird 91

### DISCUSSION THREAD

**Discussion by beefstew** [Extended-Thunder/send-later#361] [Q&A] [answered]
Thunderbird 91.0b6 (macOS 11.5.2 64-bit)
Send Later Version 8.9.5 (July 25, 2021)

The Send Later dialogue appears blank if the New Message window has been moved since it was spawned.

Steps to reproduce:
1. Ensure Thunderbird running and Send Later installed
2. ⌘N to spawn a New Message composition window
3. Click Send Later button and see:
![Screen Shot 2021-08-11 at 10 20 25 PM](https://user-images.githubusercontent.com/2262454/129128889-43293a2f-b244-4906-9c8c-cfb614836fd3.png)
4. Move/re-position the New Message window (will cause Send Later pop-up to close)
5. Click Send Later button and see: 
![Screen Shot 2021-08-11 at 10 20 36 PM](https://user-images.githubusercontent.com/2262454/129128975-1608dbda-bf3c-4ecd-ac42-6fe708d581c6.png)

Expected behavior:
Not blank, ever.

Thanks,
Chris

**Comment by jph-sendlater** [comment, marked as answer]
This is a Thunderbird bug. Send Later doesn't handle low-level operations like rendering pop-ups. The Thunderbird devs would definitely be interested to know about it, though. I think [this](https://bugzilla.mozilla.org/buglist.cgi?product=Thunderbird&component=OS%20Integration) would be the correct place to report that bug.

**Reply by beefstew** [reply]
~https://bugzilla.mozilla.org/show_bug.cgi?id=1725302~

https://bugzilla.mozilla.org/show_bug.cgi?id=1710608

**Comment by beefstew** [comment]
Duplicate of #333 and #328
